### DISCUSSION TITLE
Dark mode and functions

### DISCUSSION THREAD

**Discussion by n-ham** [pocketbase/pocketbase#2659] [Q&A] [answered]
Curious whether there's any plans for dark mode option with the dashboard? It looks nice using the darkreader extension, but would be ace to be able to configure it to be dark mode in the dashboard, possibly even for the admin login and everything. Also are there any plans to add in functions?

**Comment by ganigeorgiev** [comment, marked as answer]
> Curious whether there's any plans for dark mode option with the dashboard? 

Dark mode for now is left aside because at this stage it will be hard to maintain (_you can subscribe to #407_).
The Admin UI will be also most likely refactored sometime in the future in relation to #898.

> Also are there any plans to add in functions?

If you mean server-side functions like AWS Lambdas, Cloudflare Workers, etc., then No (at least not in the same sense).

If you want to extend PocketBase you can use it as a Go framework/toolkit - https://pocketbase.io/docs/use-as-framework/. With the next release there will be also an option to bind to the app hooks with JavaScript too (_via [goja](https://github.com/dop251/goja), similarly to how we currently handle the `pb_migrations` when using the prebuilt executable_).

_For more details on the planned tasks, you can check the [public roadmap](https://github.com/orgs/pocketbase/projects/2/views/1)._

**Comment by maxirozay** [comment]


**Reply by ganigeorgiev** [reply]
Please don't spam like that again!
